controllers/api/openstack/vm.py

```diff
@@ -158,6 +208,7 @@ def GET(self, action):
                 'type'     : imageName,
                 'candelete': True,
 		'keypair'  : server.key_name,
+                'aquilon'  : aquilon,
```

Comment:
again, this indentation looks 'wonky' in a side by side comparison view but not when viewing the file as a whole...